Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBaseTest.cs

```diff
@@ -113,7 +114,7 @@ public void RenderTitleCellNoSorting ()
 
       var titleSpan = Html.GetAssertedChildElement (sortCommandLink, "span", 0);
       Html.AssertAttribute (titleSpan, "id", List.Object.ClientID + "_0_Title");
-      Html.AssertTextNode (titleSpan, Column.ColumnTitleDisplayValue, 0);
+      Html.AssertTextNode (titleSpan, Column.ColumnTitleDisplayValue.ToString(), 0);
```

Comment:
```suggestion
      Html.AssertTextNode (titleSpan, Column.ColumnTitleDisplayValue.ToString(WebStringEncoding.HtmlWithTransformedLineBreaks, 0);
```

You should use `ToString(WebStringEncoding.HtmlWithTransformedLineBreaks)` instead of `ToString()`, but since it wouldn't make a difference in this test, `ToString()` might also be ok.
What do you say? @MichaelKetting

Reply:
Makes sense, I'll change it. Could easily revert it if @MichaelKetting has a different opinion.

Reply:
Okay, consens seems to be that we go with `ToString(WebStringEncoding.HtmlWithTransformedLineBreaks)`? I like this choice because it means we don't take a shortcut with a different semantic just for convenience's sake. 

Sidequest / Continuation:
We probably should add an overload for `AssertTextNode` that accepts webstring and uses `ToString(WebStringEncoding.HtmlWithTransformedLineBreaks)` under the hood because this is the default semantic of `WebString.Write`. @a-ctor ?

Reply:
Yeah sure why not